.travis.yml

```diff
@@ -5,8 +5,8 @@ os: linux
 dist: xenial
 sudo: enabled
 install:
```

Comment:
This alone line doesn't crash build?

Reply:
looks like still npm install is executed. I wanted to avoid it, I'll change config to avoid this